forge-gui/src/main/java/forge/util/ImageFetcher.java

```diff
@@ -65,13 +65,13 @@ private String getScryfallDownloadURL(PaperCard c, String face, boolean useArtCr
                     }
                 }
             }
-            return null;
         } else {
             String setCode = edition.getScryfallCode();
             String langCode = edition.getCardsLangCode();
-            return ForgeConstants.URL_PIC_SCRYFALL_DOWNLOAD +
-                    ImageUtil.getScryfallDownloadUrl(c, face, setCode, langCode, useArtCrop);
+            downloadUrls.add(ForgeConstants.URL_PIC_SCRYFALL_DOWNLOAD + ImageUtil.getScryfallDownloadUrl(c, face, setCode, langCode, useArtCrop));
```

Comment:
For most cards you would just add the same URL twice...
Meaning two threads would try the same download, which seems pretty nasty ⚠️

Reply:
Alright that should be fixed now.